ansible/roles/nginx/defaults/main.yml

```diff
@@ -516,7 +523,7 @@ nginx_default_tls_protocols: '{{ [ "TLSv1", "TLSv1.1", "TLSv1.2", "TLSv1.3" ]
 #
 # See also: https://security.stackexchange.com/questions/31772/
 # Set to ``False`` to disable ECC.
-nginx_default_ssl_curve: 'secp384r1'
+nginx_default_ssl_curve: 'secp521r1:secp384r1:prime256v1'
```

Comment:
What goes into ssl-config.mozilla.org is first brainstormed on a wiki page. It is linked (somewhat inconspicuously) in the footer of ssl-config. https://wiki.mozilla.org/Security/Server_Side_TLS mentions:

> TLS curves: X25519, prime256v1, secp384r1

Indeed I must have copied the `secp521r1:secp384r1:prime256v1` from the wrong place, will fix it to include secp384r1, secp256r1, x448, x25519 and prime256v1.

Reply:
After a bit more digging I found there's some concern over the rigidity of `prime256v1` because it's a NIST curve. See here for the rabbit hole: https://github.com/nodejs/node/issues/1495

I'm now removing `prime256v1` from the debops list. Just NCSC-NL recommended curves remain.

Reply:
Openssl on Debian Buster only supports one of the four NCSC-NL curves: `secp256r1` (see `openssl ecparam -list_curves`). Configuring just the one curve does not seem to limit availability (checked SSLLabs and two physical devices). One curve it is.